frontend/public/style/_vars.scss

```diff
@@ -19,26 +19,13 @@ $fa-font-path: $consoleWebDistPath + '/';
 // Use PatternFly 4 font family in PatternFly 3.
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
 $font-size-base: 14px;
-// Use PatternFly 4 heading font sizes in Patternfly 3.
-$font-size-h1: var(--pf-global--FontSize--2xl);
-$font-size-h2: 20px; // must be literal value because of calculation in patternfly/dist/sass/patternfly/_list-view.scss
-$font-size-h3: var(--pf-global--FontSize--lg);
-$font-size-h4: var(--pf-global--FontSize--md);
-$font-size-h5: var(--pf-global--FontSize--md);
-$font-size-h6: var(--pf-global--FontSize--md);
-// Patternfly 3 defaults to 40px, which leaves a lot more whitespace than we're used to.
 $grid-gutter-width: 30px;
```

Comment:
This is no longer needed since the Bootstrap 3 default is 30px and we no longer include the PatternFly override.

Reply:
removed